.github/actions/github-release/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "wasmtime-github-release",
```

Comment:
wasmtime ? :)

Reply:
Yeah, i think it doesn't make sense to fix all various references to wasm